File: Server (ASP.NET Core)/Petadmin/Models/FollowUpPage.cs
using System.Collections.Generic;
using Petadmin.Core.Models;
using PetAdmin.Core.Models;

namespace Petadmin.Models
{
    public class FollowUpPage
    {
        public Animal Animal { get; set; }
        public Owner Owner { get; set; }
        public List<FollowUp> FollowUpsList { get; set; }
    }
}

File: Client (Angular)/models/followup-page.ts
import {Owner, OwnerEntity} from './owner';
import {Animal, AnimalEntity} from './animal';
import {Followup} from './followup';

export interface FollowupPage {
  animal: Animal;
  owner: Owner;
  followUpsList: Followup[];
}

export class FollowupPageEntity implements FollowupPage{
  constructor() {
    this.animal = new AnimalEntity();
    this.owner = new OwnerEntity();
    this.followUpsList = [];
  }
  animal: Animal;
  owner: Owner;
  followUpsList: Followup[];
}
